### ISSUE TITLE
'npm install "package-name@>=version"' redirect npm install output to file named 'version'

### ISSUE THREAD

**Issue by robert-matusewicz** [PowerShell/PowerShell#17827]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you can repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

I think I found a problem installing the npm package with at least X version in PowerShell.

When you run this command in PowerShell 7.2.5 or PowerShell 7.3.0-preview.6:
`npm install "express@>=5.0.0-beta.1"`

Instead of installing express with at least 5.0.0-beta.1 version installs the latest stable version (4.18.1) and redirects the output to a file named "5.0.0-beta.1"

I also tested that in cmd, and it works as expected - express@5.0.0-beta.1 is installed.

### Expected behaviour

```console
`npm install "express@>=5.0.0-beta.1"` should install express@5.0.0-beta.1 or newer version
```

### Actual behaviour

```console
`npm install "express@>=5.0.0-beta.1" installs the latest stable version of express (express@4.18.1 at the time of writing this text)
`npm install "express@>=5.0.0-beta.1" creates a file "5.0.0-beta.1" with the content of npm install output
```

### Error details

```console
In this scenario, PowerShell does not return any error
```

### Environment data

```powershell
Name                           Value
----                           -----
PSVersion                      7.2.5
PSEdition                      Core
GitCommitId                    7.2.5
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

For preview:

Name                           Value
----                           -----
PSVersion                      7.3.0-preview.6
PSEdition                      Core
GitCommitId                    7.3.0-preview.6
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by robert-matusewicz** [comment]
I think I was too fast with raising the issue here. Yarn install works as expected.

`yarn add express@>=5.0.0-beta.1`

installs express@5.0.0-beta.1

**Comment by robert-matusewicz** [comment]
Closing at that was npm issue. Seems to be fixed in npm@18.5.1
